core/src/main/kotlin/org/evomaster/core/search/gene/NumericStringGene.kt

```diff
@@ -0,0 +1,114 @@
+package org.evomaster.core.search.gene
+
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.search.impact.impactinfocollection.value.OptionalGeneImpact
+import org.evomaster.core.search.impact.impactinfocollection.value.numeric.NumericStringGeneImpact
+import org.evomaster.core.search.service.AdaptiveParameterControl
+import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.mutator.MutationWeightControl
+import org.evomaster.core.search.service.mutator.genemutation.AdditionalGeneMutationInfo
+import org.evomaster.core.search.service.mutator.genemutation.SubsetGeneSelectionStrategy
+import java.math.BigDecimal
+
+class NumericStringGene(
```

Comment:
explain this gene. write something like this is a number in all aspects, but in its phenotype it is going to be outputted as a string